tests/net/tx_timestamp/src/main.c

```diff
@@ -323,6 +323,11 @@ static void iface_cb(struct net_if *iface, void *user_data)
 	    net_if_get_by_iface(iface));
 
 	if (net_if_l2(iface) == &NET_L2_GET_NAME(ETHERNET)) {
+		if (ud->eth_if_count >= ARRAY_SIZE(eth_interfaces)) {
+			DBG("Invalid interface %p\n", iface);
```

Comment:
Is this DBG() right one? This looks like an error.

Reply:
This is not an error actually, depending on setup there might be one extra network interface.